src/App.css

```diff
@@ -45,6 +45,6 @@ img {
     left: 50px;
 }
 
-.sprite__leftImage {
+.sprite__left-img {
     transform: scaleX(-1);
```

Comment:
Nice. guese this is for inverting images right to left?

Reply:
Yes, for inverting images right to left for the player, only necessary for the bottom left sprite